frame/assets/src/lib.rs

```diff
@@ -230,11 +230,11 @@ decl_event! {
 		<T as Trait>::Balance,
 		<T as Trait>::AssetId,
 	{
-		/// Some assets were issued.
+		/// Some assets were issued. [asset_id, origin, total_supply]
```

Comment:
The second value is clearly not the origin.

Reply:
sender? I was just checking a references. Found this
```
Self::deposit_event(RawEvent::Issued(id, origin, total));
```

Reply:
or just `account` I think would be enough.

Reply:
`owners` makes more sense to me.